packages/ui/test/membership/modals/UpdateMembershipModal.test.tsx

```diff
@@ -67,42 +80,57 @@ describe('UI: UpdatedMembershipModal', () => {
     batchTx = stubTransaction(api, 'api.tx.utility.batch')
     stubTransaction(api, 'api.tx.members.updateProfile')
     profileTxMock = api.api.tx.members.updateProfile as unknown as jest.Mock
-    renderModal()
+    useMyMemberships.members = [getMember('alice'), getMember('bob')]
+    useMyMemberships.setActive(getMember('bob'))
   })
 
   it('Renders a modal', async () => {
+    act(() => {
+      renderModal()
+    })
     expect(await screen.findByText('Edit membership')).toBeDefined()
   })
 
   it('Is initially disabled', async () => {
+    act(() => {
+      renderModal()
+    })
     expect(await getButton(/^Save changes$/i)).toBeDisabled()
   })
 
   it('Enables button on external resources change', async () => {
-    expect(await getButton(/^Save changes$/i)).toBeDisabled()
-
-    await act(async () => {
-      fireEvent.change(screen.getByTestId('twitter-input'), { target: { value: 'joystream@mail.com' } })
+    act(() => {
+      renderModal()
     })
+    expect(await getButton(/^Save changes$/i)).toBeDisabled()
+    fireEvent.change(screen.getByTestId('twitter-input'), { target: { value: 'joystream@mail.com' } })
 
     expect(await getButton(/^Save changes$/i)).toBeEnabled()
   })
 
   it('Enables button on member field change', async () => {
+    act(() => {
+      renderModal()
+    })
```

Comment:
Same for above, this should work too:
```suggestion
    act(renderModal)
```